src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -8,6 +8,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NoWarn>$(NoWarn),649</NoWarn>
     <DefineConstants Condition="'$(TargetGroup)' == 'uap'">$(DefineConstants);uap</DefineConstants>
```

Comment:
'uap' define does not appear to be used anywhere in this project. Should it be deleted?

(If it was used, you would likely need to define it for uapaot as well or something.)